viscy/data/typing.py

```diff
@@ -88,3 +88,28 @@ class TripletSample(TypedDict):
     positive: NotRequired[Tensor]
     negative: NotRequired[Tensor]
     index: NotRequired[TrackingIndex]
+
+
+# NOTE: these are the only columns that are allowed for the annotation dataframe.
+AnnotationColumns = Literal[
+    "infection_state", "cell_cycle_phase", "cell_remodeling_state"
+]
+
+
+# NOTE: The following labels are not mutable. They are used to map the labels to the integer values.
+class InfectionStateLabel(dict):
+    uninfected: 0
+    infected: 1
+    unknown: -1
```

Comment:
Class attributes of the dict type are not the same as key-values in dict. Use concrete objects instead:

```suggestion
INFECTION_STATE_LABEL = {
    "uninfected": 0
    "infected": 1
    "unknown": -1
    }
```

Reply:
Also since these are not types, they should be in a separate module.